src/test/java/io/aiven/kafka/connect/http/sender/OAuth2HttpSenderTest.java

```diff
@@ -121,6 +122,39 @@ void buildAccessTokenAuthHeaderFromCustomSettings(@Mock final HttpResponse<Strin
         );
     }
 
+    @Test
+    void buildSpecifiedContentType(@Mock final HttpResponse<String> accessTokenResponse)
+            throws IOException, InterruptedException {
+        final var config = new HashMap<>(defaultConfig());
+        config.put("oauth2.client.authorization.mode", "url");
+        config.put("oauth2.client.scope", "a,b,c");
+        config.put("oauth2.response.token.property", "some_token");
+        config.put("http.headers.content.type", "application/json");
```

Comment:
Similar nit-pick in here. Could you extract `"application/json"` here and in line 155 in its own variable?

Reply:
Removed magic string